CONTRIBUTING.md

```diff
@@ -1,104 +1,104 @@
-  - **Dimensions**:
-    - Height: 512 pixels
-    - Width: Auto (maintaining aspect ratio)
-  - **Transparency**: Enabled
+### Format Requirements
 
-### Cropping
+- **SVG Format**: All icons must be submitted in SVG format
+- **Auto-Generated Formats**: PNG and WEBP versions are generated automatically with:
+  - Height: 512 pixels
+  - Width: Auto (maintaining aspect ratio)
+  - Transparency: Enabled
 
-- **Remove Empty Space**: Crop any empty space from your SVG files to ensure the icon is properly centered and sized. You can use [SVG Crop](https://svgcrop.com/) to assist with this.
+### Quality Standards
 
-### Light and Dark Versions
+- **Clean SVG**: No embedded raster images in SVG files
+- **Proper Cropping**: Remove empty space for proper centering
+  - Use [SVG Crop](https://svgcrop.com/) for assistance
+- **No Upscaling**: Maintain original quality without artificial enlargement
 
-- **Monochrome or Single Primary Color Icons**:
-  - If your icon is monochrome, please provide additional versions if applicable:
-    - **`-light` Version**: For icons primarily dark or using black as a main color, provide a `-light` version for light backgrounds.
-    - **`-dark` Version**: For icons primarily light or using white as a main color, provide a `-dark` version for dark backgrounds.
-  - **Examples**:
-    - A black logo should include a `-light` version where black is inverted.
-    - A multicolored logo using black should provide a `-light` version with the black replaced.
-  - **Tool Recommendation**: [DEEditor](https://deeditor.com/) can help adjust icon colors if needed.
+### Light & Dark Variants
 
-### File Naming
+For monochrome or single-color icons:
 
-- **Kebab Case**: Name your files using kebab case (lowercase words separated by hyphens). For example, "Nextcloud Calendar" becomes `nextcloud-calendar.svg`.
-  - **Note**: Filenames are automatically converted to kebab case, but please double-check your naming to avoid conflicts or errors.
+- **Light Variant**: Required for dark backgrounds
+  - Invert black elements
+  - Adjust colors for visibility
+- **Dark Variant**: Required for light backgrounds
+  - Invert white elements
+  - Adjust colors for visibility
 
-### Quality Requirements
+**Tool Recommendation**: [DEEditor](https://deeditor.com/) for color adjustments
 
-- **No Upscaled Images**: Icons should maintain their original quality without artificial enlargement.
-- **No Embedded Raster Images in SVGs**: Ensure that SVG files are true vector graphics without embedded raster images.
-
-## Git Commit Messages
+### File Naming
 
-- **Use Semantic Commits**: Follow the format <type>(scope): description:
-  - `feat(icons): add nextcloud-calendar` when adding new icons.
+- **Kebab Case**: Use lowercase with hyphens
+  - Example: "Nextcloud Calendar" → `nextcloud-calendar.svg`
+- **Variant Suffixes**:
+  - `-light` for dark backgrounds
+  - `-dark` for light backgrounds
```

Comment:
it doesn't make sense to tell users to add 
-light for light, since we match on the base icon (`sonarr`), it should be nothing for light and -dark for dark mode (`sonarr` and `sonarr-dark`)
if the base icon is dark mode then nothing for dark and -light for dark (`sonarr`, `sonarr-light`)